main.go

```diff
@@ -145,23 +147,23 @@ func handleGetDebug() http.Handler {
 	return mux
 }
 
-// handleGetOpenapi returns an [http.HandlerFunc] that serves the OpenAPI specification YAML file.
+// handleGetOpenAPI returns an [http.HandlerFunc] that serves the OpenAPI specification YAML file.
 // The file is embedded in the binary using the go:embed directive.
-func handleGetOpenapi(version string) http.HandlerFunc {
-	body := bytes.Replace(openapi, []byte("${{ VERSION }}"), []byte(version), 1)
-	return func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Set("Content-Type", "text/plain")
+func handleGetOpenAPI(version string) http.HandlerFunc {
+	body := bytes.Replace(openAPI, []byte("${{ VERSION }}"), []byte(version), 1)
+	return func(w http.ResponseWriter, _ *http.Request) {
+		w.Header().Set("Content-Type", "application/vnd.oai.openapi+json")
```

Comment:
```suggestion
		w.Header().Set("Content-Type", "application/yaml")
```

looks like this media type is not official yet. and i want to be simple, intuitive content type rather than strict type
 
https://www.iana.org/assignments/media-types/media-types.xhtml